config/protocol.mainnet.neofs.yml

```diff
@@ -27,6 +27,7 @@ ProtocolConfiguration:
     Aspidochelone: 3000000
     Basilisk: 4500000
     Cockatrice: 5800000
+    Domovoi: 5900000
```

Comment:
~10 days from the current 5838334. Is it enough or we need a smaller value?

Reply:
Doesn't really matter, can be 5800000 as well.

Reply:
Fixed.